src/main/java/io/openliberty/tools/intellij/lsp4jakarta/lsp4ij/jsonb/JsonbDiagnosticsCollector.java

```diff
@@ -120,22 +121,22 @@ private void collectJsonbTransientAccessorDiagnostics(PsiJavaFile unit, PsiClass
                 // or if @JsonbTransient is not mutually exclusive
                 if (hasFieldConflict || hasJsonbAnnotationOtherThanTransient(jsonbAnnotationsForAccessor))
                     createJsonbTransientDiagnostic(unit, diagnostics, accessor, jsonbAnnotationsForAccessor,
-                            JsonbConstants.ERROR_MESSAGE_JSONB_TRANSIENT_ON_ACCESSOR);
+                            JsonbConstants.DIAGNOSTIC_CODE_ANNOTATION_TRANSIENT_ACCESSOR);
 
             }
         }
         if (createDiagnosticForField)
             createJsonbTransientDiagnostic(unit, diagnostics, field, jsonbAnnotationsForField,
-                    JsonbConstants.ERROR_MESSAGE_JSONB_TRANSIENT_ON_ACCESSOR);
+                    JsonbConstants.DIAGNOSTIC_CODE_ANNOTATION_TRANSIENT_ACCESSOR);
```

Comment:
Similar here, should this be `DIAGNOSTIC_CODE_ANNOTATION_TRANSIENT_FIELD` - again just inferring based on params and other usage - just double checking

Reply:
In this case transient is on the accessor and there is a diagnostic on the field warning about the accessor.